src/xml.tsx

```diff
@@ -174,6 +174,7 @@ export class SvgFromXml extends Component<XmlProps, XmlState> {
     try {
       this.setState({ ast: xml ? parse(xml) : null });
     } catch (e) {
+      this.props.onError?.(e);
       console.error(e);
```

Comment:
@WoLewicki `console.error(e);` does not necessarily need to operate by default.

```suggestion
    this.props.onError ? this.props.onError(e) : console.error(e);
```

Is it correct to say this way? That seems like a good approach!

Reply:
Isn't it the same as the one I mentioned: https://github.com/software-mansion/react-native-svg/commit/3c32a6fdd580dea53ecc271288155117ce040b55#diff-516de011a591b7585d24e2a1ee084a41d09bfc6c9ba70b1320e7e39bebd792f7R131 ? You either have your own implementation or use the `console.error`. Or am I missing something?

Reply:
@WoLewicki 
The commits you mentioned are for `SvgXml` and `SvgUri` components.
The parts I modified are `SvgFromUri` and `SvgFromXml`, so they are different.
And apparently `OnError` doesn't work when I use `SvgFromUri`.